src/style.css

```diff
@@ -2,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form__input {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  padding: 0;
+  margin: 20px 0;
```

Comment:
Be consistent with your margins. You've added both top and bottom margins here. Try to stick to either top or bottom to avoid potential margin collapse.